### DISCUSSION TITLE
Should sceneform be replaced with OpenGL on Android?

### DISCUSSION THREAD

**Discussion by CariusLars** [CariusLars/ar_flutter_plugin#13] [Ideas]
Moved here from #12 :

Open questions: As Sceneform support was dropped by Google, should the Android implementation of this plugin move to another renderer?

**Comment by CariusLars** [comment]
The community-supported version of Sceneform is updated to include new features like Depth API (https://github.com/ThomasGorisse/sceneform-android-sdk/releases/tag/v1.19.1), Google's own example apps now use OpenGL as a rendere, which is more flexible but also much more complex to implement (https://codelabs.developers.google.com/codelabs/arcore-depth#0)

Both options are worth looking into, I'm open for ideas and opinions where to go with this!

**Reply by sameeerkashyap** [reply]
@CariusLars If the community scene form is actively maintained then it might be worth going ahead. Since OpenGL render would be too complex to implement ( unless it provides significant features and improvements). 

and if you could open specific issues or tell me what features need to be implemented I can do some research and help you out.

**Reply by CariusLars** [reply]
Hey @Sameerkash , thanks for your interest in contributing, that's awesome! I've created issues for three tasks that I think are worth looking into, namely migrating to the community-supported Sceneform (#16), adding image anchor functionality (#17) and adding face anchor functionality (#18). Looking forward to your ideas!
